charts/kafka-lag-exporter/values.yaml

```diff
@@ -69,7 +69,15 @@ akkaLogLevel: DEBUG
 image:
   repository: lightbend/kafka-lag-exporter
   tag: 0.5.5-SNAPSHOT
+  # If digest is set it will be used instead of tag to specify the image
+  # digest: sha256:0f6387aa011e6eb7952ea211ac139bf8613ad3ef6954a1a5d910676d293bd610
   pullPolicy: Always
+securityContext:
```

Comment:
Is there a reason you left this as a default?  I suspect it will work in most cases, but I think I would prefer it be commented out.

Reply:
No reason other than I don't like running the pods with more privileges than necessary. But most important is that it's configurable, so I've changed the chart to have an empty default value.